### ISSUE TITLE
Steady sine tone - after playing/rendering

### ISSUE THREAD

**Issue by csounder** [csound/web-ide#102]
When I click on a featured example to play - such as Thorin Kerr's - Blue and then... stop playing, what emerges is a content tone (like a final buffer that is looping).  This happens when I stop playing any example.

**Comment by kunstmusik** [comment]
I was able to reproduce this on Chrome but not on other browsers. Interestingly, it seems to be an issue on Chrome 84 (the latest stable release), but not on Chrome 86 (the latest Canary dev release). I wonder if a bug was introduced in Chrome 84 that was later fixed?  We'll have to check the Chrome bug tracker to find out.

**Comment by kunstmusik** [comment]
Following up, we found that this was a bug specifically in Chrome 84. This was reported and triaged in the Chromium bug tracker:

https://bugs.chromium.org/p/chromium/issues/detail?id=1106973&q=audioworklet&can=2
https://bugs.chromium.org/p/chromium/issues/detail?id=1099756

The bug reports mention that it is fixed in Chrome 85. I saw another report of a problem on the WebAudio Slack, also with 84, so it is a Chrome problem.  We can't really fix the issue since it's in the system, but we might be able to put up a dialog for the short term to report to the user that Chrome 84 has a bug and to use another browser.
